src/actions/index.js

```diff
@@ -11,8 +14,31 @@ export const DECREMENT = 'DECREMENT';
 
 export const increment = () => {
   // Fill in this function
+  return {
+    type: INCREMENT
+  }
 };
 
 export const decrement = () => {
-  // Fill in this function
+  return {
+    type: DECREMENT
+  }
 };
+
+export const reset = () => {
+  return {
+    type: RESET
+  }
+}
+
+export const incrementOdd = () => {
```

Comment:
So you technically don't need either of these action creators. You can just do logic inside the functions inside the Counter component and then re-use increment.